src/txn/txn_recover.c

```diff
@@ -989,6 +994,18 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
         __wt_err(session, ret, "Recovery failed");
     }
 
+    /*
+     * Close all the cursors and open dhandles if rollback to stable was called to make sure that
+     * all the modified pages as part of RTS are evicted from the cache.
+     *
+     */
+    if (rollback_to_stable_called) {
```

Comment:
I feel we should close all handles irrespective of whether RTS is called or not as the base write gen increment is moved to a location where it is possible that handles may be opened before incrementing the write generation. So no need of any rollback flag.

Reply:
Fixed